components/style/servo_config_facade.rs

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ /* Ref: https://github.com/servo/servo/issues/15460 for more info for why it's needed. */
+
+// TODO: write docs for this mod
+use servo_config;
```

Comment:
aah, that explains it. I think ill add an cfg. Just curious, how did you debug that because error did not log anything